libutils/src/zf_log.c

```diff
@@ -1,13 +1,7 @@
 /*
- * Copyright 2017, Data61
- * Commonwealth Scientific and Industrial Research Organisation (CSIRO)
- * ABN 41 687 119 230.
+ * Copyright 2017, Data61, CSIRO (ABN 41 687 119 230)
```

Comment:
This file is minimal Data61 code and predominantly from here (MIT): https://github.com/wonder-mice/zf_log/blob/master/LICENSE

Reply:
Same as zf_log.h

Reply:
Cool, thanks, will update.